python/guider.py

```diff
@@ -158,6 +163,7 @@ def __init__(self, hostname = "localhost", instance = 1):
         self.accum_dec = _Accum()
         self.Stats = GuideStats()
         self.Settle = None
+        self.eventListener = None if eventListener is None else eventListener
```

Comment:
how about this (equivalent and simpler):
```suggestion
        self.eventListener = eventListener
```